app/facets/participant/participant.coffee

```diff
@@ -12,12 +12,21 @@ class Participant
 				id: @id
 				display_name: user.displayName
 
-		@game_session_id = new nx.Cell
 
 		@game_session = new nx.Cell
-			'<-': [@game_session_id, (id) => @service.game_sessions.get id]
 			action: (game_session) =>
-				game_session.add_participant @
+				game_session?.add_participant @
+
+		@game_session_id = new nx.Cell
+			'->': [
+				((id) =>
+					if @service.game_sessions.has id
+						@game_session
+					else
+						[]
```

Comment:
That's not a default value but sync target, that is, if there's a session then the target is `game_session` cell, otherwise there are no sync targets (`undefined` would be better, but currently nexus understands only cells or arrays of cells, hence `[]`)

SO UGLY :crying_cat_face: